docs/image-promotion.md

```diff
@@ -280,4 +280,4 @@ check to occur.
 [k/k8s.io]: https://git.k8s.io/k8s.io
```

Comment:
I'm not sure why https://git.k8s.io/k8s.io/k8s.gcr.io/Vanity-Domain-Flip.md#prow-integration was not duplicated in  https://git.k8s.io/k8s.io/registry.k8s.io .

Reply:
Because we're already promoting to both backends simultaneously. We'll stop promoting to k8s.gcr.io backends soon but that's been on pause with the redirect rolling out. More discussion in k8s infra and the associated KEP

Reply:
The vanity domain flip is also a historical document for a different process involving GCR => GCR. This time it's GCR => Kubernetes DIY Alias in front of AR,S3